src/app/shared/components/logo/logo.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="logo" routerLink="/">
+    <img src="assets/images/youtube.svg" alt="youtube logo" />
+</div>
```

Comment:
**you don't need to fix it**

not sure that in the scope of this project we need to create a separate component for a logo. But if it is a big project with different configurations (for ex. add title for some cases) it makes sense